setup.py

```diff
@@ -35,11 +35,14 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        'django>=3.0,<3.2',
+        'django>=3.0,<=4.0rc1',
```

Comment:
What about:
```suggestion
        'django>=3.0.0,<4.1.0',
```
So that we include later versions of Django 4 once released?